html-css/week1/homework01/index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&family=Barlow:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Abril+Fatface&display=swap" rel="stylesheet">
+    <title>Soheib Ider Portfolio</title>
+</head>
+<body>
+    <div class="background">
+        <div class="top">
+            <h1 id="opening">Welcome</h1>
+        </div>
+        <header>    
+            <div class="navwrapper">
+                <div class="navcard"><a href="#about">About</a></div>
+                <div class="navcard"><a href="#skills">Skills</a></div>
+                <div class="navcard"><a href="#portfolio">Portfolio</a></div>
+                <div class="navcard"><a href="#contact">Contact</a></div>
+            </div>
+        </header>
+        <section class="contwrapper">
+           <div class="contcard" id="about">
+            <p>I'm <b id="name">Soheib</b></p>
+                <p><img id="profilepic" src="./assets/577A1074 4.jpg" alt="profile picture">
```

Comment:
Bad value ./assets/577A1074 4.jpg for attribute src on element `<img>` Illegal character in path segment: space is not allowed.

Reply:
Thank you very much for the advice. I have implemented the change